wcomponents-core/src/main/java/com/github/bordertech/wcomponents/util/Util.java

```diff
@@ -129,4 +131,19 @@ public static String leftTrim(final String aString) {
 		return aString.substring(start);
 	}
 
```

Comment:
What if this was included in the FileValidationUtil? Which means maybe FileValidationUtil might need to be named a bit more generic like FileUtil.